SSA/Projects/FullyHomomorphicEncryption/Basic.lean

```diff
@@ -632,7 +646,7 @@ theorem R.representative_fromTensor_eq_fromTensor' (tensor : List Int) :
     R.representative q n (R.fromTensor tensor) =
       R.representative' q n (R.fromTensor' q tensor)  %ₘ (f q n) := by
   simp only [representative]
-  rw [fromTensor_eq_fromTensor'_fromPoly];
+  simp [fromTensor_eq_fromTensor'_fromPoly ];
```

Comment:
interesting, what happened here, did `rw` break? do you know why? (and why is there an extra whitespace at the end of the theorem name)?